tests/migrator/test_books.py

```diff
@@ -1059,48 +1059,79 @@ def test_extensions(app):
                 <subfield code="i">no longer applicable</subfield>
                 <subfield code="p">Expert ICS-25.160</subfield>
                 <subfield code="v">withdrawn</subfield>
-                <subfield code="z">Reviewed in December 2019</subfield>
+                <subfield code="z">20040621</subfield>
             </datafield>
             """,
             {
                 "extensions": {
-                    "standard_review_applicability": "no longer applicable",
+
+                    "standard_review_applicability": ["NO_LONGER_APPLICABLE"],
                     "standard_review_validity": "withdrawn",
-                    "standard_review_checkdate": "Reviewed in December 2019",
+                    "standard_review_checkdate": "2004-06-21",
                     "standard_review_expert": "Expert ICS-25.160",
                 },
             },
         )
+        with pytest.raises(UnexpectedValue):
+            check_transformation(
+                """
+                <datafield tag="925" ind1=" " ind2=" ">
+                    <subfield code="i">no longer applicable</subfield>
+                    <subfield code="p">Expert ICS-25.160</subfield>
+                    <subfield code="v">withdrawn</subfield>
+                    <subfield code="z">Reviewed in December 2019</subfield>
+                </datafield>
+                """,
+                {
+                    "extensions": {
+                        "standard_review_applicability": "",
+                        "standard_review_validity": "withdrawn",
+                        "standard_review_checkdate": "Reviewed in December",
+                        "standard_review_expert": "Expert ICS-25.160",
+                    },
+                },
+            )
         check_transformation(
             """
             <datafield tag="693" ind1=" " ind2=" ">
-                <subfield code="a">CERN LHC</subfield>
+                <subfield code="a">CERN SPS</subfield>
                 <subfield code="e">ATLAS</subfield>
             </datafield>
             <datafield tag="693" ind1=" " ind2=" ">
-                <subfield code="a">CERN LHC</subfield>
+                <subfield code="a">CERN SPS</subfield>
                 <subfield code="e">CMS</subfield>
                 <subfield code="p">FCC</subfield>
             </datafield>
             <datafield tag="925" ind1=" " ind2=" ">
                 <subfield code="i">no longer applicable</subfield>
                 <subfield code="p">Expert ICS-25.160</subfield>
                 <subfield code="v">withdrawn</subfield>
-                <subfield code="z">Reviewed in December 2019</subfield>
+                <subfield code="z">20040621</subfield>
             </datafield>
             """,
             {
                 "extensions": {
-                    "unit_accelerator": ["CERN LHC"],
+                    "unit_accelerator": ["CERN_SPS"],
```

Comment:
shouldn't be only `SPS`?

Reply:
I kept it as `CERN SPS` because we got the values from the [accelerators KB](https://cds.cern.ch/kb/export?kbname=Accelerators).

Reply:
yes I see but the goal was to separate the accelerators/experiments from institution, so it is an inconsistency with the rest

Reply:
Ok, I will remove `CERN` from the accelerators and leave only `SPS`. Thanks!